releasenotes/notes/python-3-8-c66cde69f972eb1e.yaml

```
@@ -0,0 +1,13 @@
+# Each section from every release note is combined when the
+# CHANGELOG.rst is rendered. Therefore, the text should be worded in a way
+# that does not depend on any information available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Upgrade embedded Python 3 to 3.8.1. Link to Python 3.8 changelog: https://docs.python.org/3/whatsnew/3.8.html
+
+    Note that the Python 2 version shipped in Agent v6 continues to be version 2.7.17 (unchanged). Agent v7 will ship Python 3.8.1
```

Let's be either super explicit (see below):

```suggestion
    Note that the Python 2 version shipped in Agent v6 continues to be version 2.7.17 (unchanged). Non-containerized packages of Agent v6 also ship Python 3 at version 3.8.1, and Agent v7 only ships Python 3.8.1.
```

or we can simply remove the `Agent v7 will ship Python 3.8.1` sentence, since I feel it can be interpreted as `only Agent v7 ships Python 3.8.1`. Up to you

---

Fair enough!